corehq/apps/hqwebapp/models.py

```diff
@@ -43,6 +43,21 @@ class Meta(object):
     def html(self):
         return mark_up_urls(self.text)
 
+    @property
+    def status(self):
```

Comment:
Okay, coming back after reviewing the rest of the PR, and I get it now - it's to make scheduled alerts still toggle-able.

Another option for this design that I thought of during review is to drop the task entirely and instead modify the query in `MaintenanceAlert.get_active_alerts`.  That is, an alert is considered active if `active=True` AND `start_time <= now < end_time`.  I think that'd result in a simpler overall design (no task needed, simpler logic about how to turn on/off alerts). What do you think?  Sorry for not thinking of this approach earlier.

Reply:
I considered moving it into `get_active_alerts` but assumed there was some reason not to do it that way, because it seemed simpler. Does that `@quickcache` decorator have the effect of making sure it's not run too often (i.e., once every minute like the task would be)?

Reply:
`@quickcache` does do that - the timeout is currently set to `60 * 60`, so it should currently recompute every hour, which I'd guess is too long a cache for this use-case, you could probably switch that to a minute or whatever if you went that route.  The cache is currently cleared any time a change is made to maintenance alerts (see the `save` method), so the long timeout is fine, but obviously scheduled alerts change the equation a bit.

Reply:
I have this set to `5 * 60` now but I'll see if support wants greater precision for any reason.